### PR TITLE
Spin: lock scroll before drag event on widget

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1104
[Problem] During drag on Spin widget the page content is scrolling
[Solution]
 - style overflow is set to hidden on scrollable parent
  before drag event on Spin widget

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

**Review by lslachciak — COMMENTED** [review]
this works when apopbar is not extended.
but when appabr is extended double scrolling occurs,
we may address this in separate issue if you want
?

File: src/js/core/widget/core/Spin.js [hunk 1]
```diff
@@ -107,9 +111,12 @@
...
 					dragTarget: "document" // "document" | "self"
 				};
-				this._ui = {};
-				this.length = this.options.max - this.options.min + 1;
-				this._prevValue = null; // this property has to be "null" on start
+				self._ui = {
+					scrollableParent: null
+				};
+				self.length = self.options.max - self.options.min + 1;
+				self._prevValue = null; // self property has to be "null" on start
```
Comment (lslachciak) [inline src/js/core/widget/core/Spin.js:118]: where _prevValue is used?

**Comment by TomaszLukawskiSam** [comment]
> this works when apopbar is not extended.
> but when appabr is extended double scrolling occurs,
> we may address this in separate issue if you want
> ?

I'll try to fix it before merge this change.

**Comment by TomaszLukawskiSam** [comment]
> this works when apopbar is not extended.
> but when appabr is extended double scrolling occurs,
> we may address this in separate issue if you want
> ?

The issue with AppBar require changes in AppBar widget. We need possibility to lock AppBar expanding during Spin drag. I think separate issue is good idea.

**Review by lslachciak — APPROVED** [review]

**Comment by lslachciak** [comment]
I created separate issue https://github.com/Samsung/TAU/issues/1111

**Pull request merged**